java/test/jmri/jmrit/logix/WarrantFrameTest.java

```diff
@@ -134,6 +158,4 @@ public void tearDown() {
         JUnitUtil.tearDown();
     }
 
-    // private final static Logger log = LoggerFactory.getLogger(WarrantFrameTest.class);
```

Comment:
No need to change, but in the future it's good to keep the commented logger. It's been more than one time there I have done some addition and things stops to work. In that case I need logging and if it's there but commented, it's easy to activate it and then deactivate it when I'm finished.